client/src/components/WordCloudModal/WordCloudModal.js

```diff
@@ -0,0 +1,58 @@
+import Modal from 'react-modal';
+import React, { useState } from 'react';
+import ReactWordcloud from 'react-wordcloud';
+
+import generateWordCloudValues from './GenerateWordCloudValues';
+
+Modal.setAppElement('#root');
+
+const wordCloudOptions = {
+    colors: ['#1f77b4', '#ff7f0e', '#2ca02c', '#d62728', '#9467bd', '#8c564b'],
+    enableTooltip: true,
+    deterministic: false,
```

Comment:
Looks like you were going for an alphabetized list, so switch the order of these :)